src/Form/UserEditType.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Form;
+
+use Bolt\Entity\User;
+use Bolt\Enum\UserStatus;
+use Bolt\Utils\LocaleHelper;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Twig\Environment;
+
+class UserEditType extends AbstractType
+{
+    /** @var LocaleHelper */
+    private $localeHelper;
+
+    /** @var Environment */
+    private $twig;
+
+    public function __construct(LocaleHelper $localeHelper, Environment $twig)
+    {
+        $this->localeHelper = $localeHelper;
+        $this->twig = $twig;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        // Create custom role options array
+        $roleOptions = [];
+        $custom_roles = $options['roles'];
+        foreach ($custom_roles as $roleName => $roleHierarchy):
```

Comment:
Ha! It's been some time since I've seen this version of the `foreach` loop. I'm surprised it even passes the tests 🙈

Reply:
why shouldn't such a foreach pass the test?
I write all my loops like that because I don't like that JS like .map() style or whatever.

Reply:
I meant this foreach:
```
foreach:
  ...
  endforeach;
```

As opposed to:

```
foreach ($foo => $bar) {

}
```

Reply:
Ah, I see.
I like to use types of "endings" because when you nest if, foreach etc. and you have a bunch of closing `}` at the end next to each other you don't know by a quick glance where you can add e.g. an else statement.

Reply:
I know it is more to write but I like more sourcecode to be more structured and readable than "compact"
Thats basically why I do that

Reply:
I'd prefer these to be in this format: 

```php
foreach (…) { 

}
```

Not because I think one is better than the other, but because of consistency. Also, PSR-12 says so: https://www.php-fig.org/psr/psr-12/#55-foreach

I'm surprised it passes tests, though. 🤔

Reply:
alright, then I will adjust that

Reply:
actually I just saw how the ecs changed that foreach.
I didn't see that till now that only the first line of code was indented.
Now with the correct { and } it looks good.